index.js

```diff
@@ -125,10 +126,18 @@ function addJs (files, ms, done) {
 
     if (Array.isArray(scripts)) {
       let userScripts = scripts.map((location) => {
-        let file = files[location]
-        if (!file.contents) return
-        let fileHash = hash(file.contents)
-        return `${location}?t=${fileHash}`
+        // Need to check if local or external
+        let fileUrl = new url.parse(location, false, true)
```

Comment:
Whoops good catch!

I first tried another module then moved to the core node one to prevent an additional dependency and accidentally left this.

Will fix as soon as I'm able!